src/Components/Blazor/WebAssembly.Authentication/src/Services/AccessTokenResult.cs

```diff
@@ -22,5 +24,24 @@ public class AccessTokenResult
         /// Gets or sets the URL to redirect to if <see cref="Status"/> is <see cref="AccessTokenResultStatus.RequiresRedirect"/>.
         /// </summary>
         public string RedirectUrl { get; set; }
+
+        /// <summary>
+        /// Determines whether the token request was successful and makes the <see cref="AccessToken"/> available for use when it is.
+        /// </summary>
+        /// <param name="accessToken">The <see cref="AccessToken"/> if the request was successful.</param>
+        /// <returns><c>true</c> when the token request is successful; <c>false</c> otherwise.</returns>
+        public bool TryGetAccessToken(out AccessToken accessToken)
+        {
+            if (string.Equals(Status, AccessTokenResultStatus.Success, StringComparison.OrdinalIgnoreCase))
```

Comment:
Why `IgnoreCase`?

Reply:
Just to be more lenient with the underlying implementations.

Reply:
Planning to move this to an enum, so won't matter.